mmv1/products/iamalpha/api.yaml

```diff
@@ -0,0 +1,121 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: IAMAlpha
+display_name: Cloud IAM
+versions:
+  - !ruby/object:Api::Product::Version
+    name: ga
+    base_url: https://iam.googleapis.com/v2alpha/
```

Comment:
Is this saying that the alpha API is considered GA? I don't believe that's something we can allow in the `google` provider even if the product is considered GA.